explorer/src/components/account/TokenLargestAccountsCard.tsx

```diff
@@ -102,8 +122,21 @@ const renderAccountRow = (
       <td>
         <Address pubkey={account.address} link />
       </td>
-      <td className="text-right">{account.uiAmount}</td>
-      <td className="text-right">{percent}</td>
+      <td className="text-right text-monospace">
+        {fixedLocaleNumber(account.uiAmount, balanceFixedPoint)}
+      </td>
+      <td className="text-right text-monospace">{percent}</td>
     </tr>
   );
 };
+
+function fixedLocaleNumber(value: number, fixedPoint: number) {
```

Comment:
Nice solution! You can also use this API: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat